addons/functions/functions/fnc_setTrenchProgress.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+/*
+* Author: nomisum
+* Animates a trench to correct progress position. Always instant.
+*
+* Arguments:
+* 0: Trench <OBJECT>
+* 1: Progress <NUMBER 0-1>
+*
+* Return Value:
+* None
+*
+* Example:
+* [TrenchObj, 0.5] call grad_trenches_functions_fnc_setTrenchProgress
+*
+* Public: No
+*/
+
+params ["_trench", "_progress", ["_duration", 0]];
```

Comment:
in the doc block above the `params`, the third parameter ("_duration") documentation is missing. Its not "always instant" anymore.

This happens easily if things are defined in two places, like in documentation block *and* in params block.

We can combine both a bit more by moving the parameter type documentation into the params block:

```suggestion
params [
    ["_trench", objNull, [objNull]],
    ["_progress", 1.0, [0]],  //  float between 0 and 1
    ["_duration", 0, [0]]
];
```